pkg/scheduler/testing/wrappers.go

```diff
@@ -417,6 +446,20 @@ func (p *PodWrapper) Label(k, v string) *PodWrapper {
 	return p
 }
 
+// MetadataLabels sets all {k,v} pair provided by `labels` to the inner pod.
+func (p *PodWrapper) MetadataLabels(labels map[string]string) *PodWrapper {
+	for k, v := range labels {
+		p = p.Label(k, v)
+	}
+	return p
+}
+
+// CreationTime sets the CreationTimestamp to `t` of the inner pod.
+func (p *PodWrapper) CreationTime(t metav1.Time) *PodWrapper {
```

Comment:
CreationTimestamp?

Reply:
`CreationTimestamp ` is a Pod attribute so cannot reuse that as a function or it will conflict.

Reply:
`p.ObjectMeta.CreationTimestamp = t`  works.

Reply:
Yes you are right, sorry, fixed now.